tests/System.Buffers.Primitives.Tests/BasicUnitTests.cs

```diff
@@ -121,13 +121,29 @@ public void ByteReadOnlySpanCtorWithRangeValidCases(byte[] bytes, int start, int
         [Fact]
         public void ByteSpanCtorWithRangeThrowsArgumentExceptionOnNull()
         {
-            Assert.Throws<ArgumentNullException>(() => { Span<byte> span = new Span<byte>(null, 0, 0); });
+            try
```

Comment:
I agree. In some scenarios, however, you cannot use DontBox and have to do this try catch (when you are not creating a new span). This is not one of those scenarios.